docs/getting-started/usage/loading-a-mnemonic.en.md

```diff
@@ -147,25 +151,24 @@ The derivation path is a sequence of numbers, or "nodes", that define the script
 
 #### Passphrase
 * ` No Passphrase `:
-Informs if the wallet has a loaded passphrase.
+Informs if the wallet has a passphrase. Adding a passphrase gives you a whole other wallet and changes the fingerprint.
```

Comment:
I'm not sure how to explain it, but something sounds odd about the phrase "...a whole other wallet", wondering if the following flows better:

"Adding or altering the passphrase results in a completely different wallet and fingerprint."

Reply:
Excellent!